cmake/CPM.cmake

```diff
@@ -354,6 +354,33 @@ function(cpm_parse_add_package_single_arg arg outArgs)
   )
 endfunction()
 
+# Check that the working directory for a git repo is clean
+function(cpm_check_working_dir_is_clean repoPath isClean)
+  # Not sure this check is necessary given git is probably key to the rest of the script
+  find_package(Git REQUIRED)
```

Comment:
Without the `find_package`, the variable `GIT_EXECUTABLE` would not be defined though - I think we can remove this comment.

Reply:
comment removed